scripts/docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
I'd put this file in `/scripts/docker/docker-entrypoint.sh` since all in `/scripts` is organised in sub-directories and no separate files.

Reply:
fixed